pygitops/operations.py

```diff
@@ -25,6 +25,7 @@ def stage_commit_push_changes(
     actor: Actor,
     commit_message: str,
     items_to_stage: Optional[List[Path]] = None,
+    kwargs_to_push: Optional[Dict] = None,
```

Comment:
I think this is actually pretty clean, and im glad that we aren't just passing the standard `kwargs` to `origin.push()` since there may be other kwargs we want to accept for other scenarios.

great design on this